### DISCUSSION TITLE
Prisma with data proxy and Postgres change type String to Json without drop DB

### DISCUSSION THREAD

**Discussion by eposha** [prisma/prisma#18490] [Q&A]
I can't find information on how to update the type without dropping the database. Can you tell me how to do this? I am using Data Proxy

Thanks

**Comment by janpio** [comment]
Change which type of what exactly?

If you want to change the field of a table in your database, you update the Prisma schema file and then migrate your database somehow.

**Reply by eposha** [reply]
I have "country" field : String. When I change it to Json type and migrate it, the console prompts me to reset the database before migration. How can I clear the country field with String type in the database and create the country field with Json type

Sorry for maybe not very accurate explanation, I'm new to this.

**Reply by janpio** [reply]
How did you get this database with the String field `country`? How did you create your schema file that you are now editing?

If you used `db pull`, you need to baseline your database first before you can use Mgirations: https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/baseline-your-database-typescript-postgres

**Reply by eposha** [reply]
@janpio Thanks, will try

**Reply by nurul3101** [reply]
@eposha If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
